tests/ptable/test_ptable_matplotlib.py

```diff
@@ -155,12 +155,12 @@ def test_return_type(self) -> None:
 def test_ptable_heatmap_splits(hide_f_block: bool) -> None:
     """Test pmv.ptable_heatmap_splits with arbitrary data length."""
     data_dict: dict[str, Any] = {
-        elem.symbol: [
-            random.randint(0, 10)  # random value for each split
```

Comment:
Not sure why this triggered [S311 (Checks for uses of cryptographically weak pseudo-random number generators.)](https://docs.astral.sh/ruff/rules/suspicious-non-cryptographic-random-usage/), might need a closer look and report this to ruff if it's a false positive